docs/guide/extending_online_editor.md

```diff
@@ -255,6 +255,85 @@ In the example above, the template files for the front end could be:
 
 Templates for Content View in the Back Office would be `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_word.html.twig` and `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_block.html.twig` respectively (assuming Admin SiteAccess uses the `admin` theme).
 
+## Custom attributes and classes
```

Comment:
let's use everywhere "data attributes", "attributes" is too generic (e.g class is also an attribute)